tools/oeedger8r/CMakeLists.txt

```diff
@@ -37,13 +36,23 @@ add_custom_command(
     DEPENDS ${OEEDGER8R_SRC}
 )
 
+if (WIN32)
 add_custom_command(
     OUTPUT ${BINARY}
-    COMMAND ${BUILD_COMMAND}
+    COMMAND C:\\windows\\system32\\WindowsPowerShell\\v1.0\\powershell -command ${CMAKE_CURRENT_SOURCE_DIR}/build.ps1 ${CMAKE_CURRENT_SOURCE_DIR} ${CMAKE_CURRENT_BINARY_DIR} 
```

Comment:
Do you need to give the full path to powershell or does just `powershell` work?

Reply:
Apparently. I could try it without. Andy was saying that it was very possible and other things are being given absolute paths.
Since that path is pretty permanent in the windows directory hierarchy. it does seem like a risk that it will change.